src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3LinuxDriver.cs

```diff
@@ -127,16 +132,16 @@ protected internal override void ClosePin(int pinNumber)
         {
             ValidatePinNumber(pinNumber);
 
-            bool isOpen = _pinModes[pinNumber] != null && _pinModes[pinNumber].InUseByInterruptDriver;
+            bool isOpen = _pinModes?[pinNumber] is object && (_pinModes[pinNumber]?.InUseByInterruptDriver ?? false);

```

Comment:
since you have a null check later this probably could be:

```suggestion
            bool isOpen = _pinModes is object && (_pinModes[pinNumber]?.InUseByInterruptDriver ?? false);
```

Reply:
Good call. That's a better pattern. Ooops. I forgot to accept your change. That wasn't intentional. Will accept next time. I very much appreciate people making "suggestions".